lib/net/http.rb

```diff
@@ -260,15 +260,20 @@ class HTTPHeaderSyntaxError < StandardError; end
   #
   #   uri = URI('https://secure.example.com/some_path?query=string')
   #
-  #   Net::HTTP.start(uri.host, uri.port,
-  #     :use_ssl => uri.scheme == 'https') do |http|
+  #   Net::HTTP.start(uri.host, uri.port, :use_ssl => true) do |http|
```

Comment:
The code snippet's URI *is* a https URI. There's no need to include that check in the snippet. Adding that check is confusing since this code snippet appears under the HTTPS heading and the context is about HTTPS in the first place.

Of course if the user wants to turn off ssl for non-https URIs, the user can do it their own way. There's no need to generalize the code snippet.